test/integration/serverTest.js

```diff
@@ -136,7 +135,7 @@ async function im21Door(doorCoreId, signal) {
         coreid: doorCoreId,
         data: `{ "data": "${signal}", "control": "86"}`,
       })
-    await helpers.sleep(10)
+    await helpers.sleep(50)
```

Comment:
Hmmm, how long do the tests take now? With all these extended `sleep` calls, it looks like this file will take like 5 minutes more to run. That feels long, doesn't it?

Reply:
They take just under 30 seconds to run on my machine

Reply:
2 minutes, 28 secs on travis

Reply:
the reason I chose this for tests is because my implementation uses "parseInt" (also our eventual db schema calls for integers in these) so the smallest threshold I can have is 1 second, for the initial timer.

Reply:
Sweet. Thanks for checking for me.